doc/conf.py

```diff
@@ -326,6 +327,7 @@
     "sparse": ("https://sparse.pydata.org/en/latest/", None),
     "cubed": ("https://tom-e-white.com/cubed/", None),
     "datatree": ("https://xarray-datatree.readthedocs.io/en/latest/", None),
+    # "opt_einsum": ("https://dgasmith.github.io/opt_einsum/", None),
```

Comment:
looks like the `opt_einsum` project is using `mkautodoc`, which doesn't seem to provide a intersphinx inventory (`mkdocstrings` would be capable of generating one). So I guess this would need interaction with that project.